mzLib/UsefulProteomicsDatabases/ProteinXmlEntry.cs

```diff
@@ -182,6 +184,38 @@ public Protein ParseEndElement(XmlReader xml, IEnumerable<string> modTypesToExcl
             return protein;
         }
 
+        internal RNA ParseRnaEndElement(XmlReader xml, IEnumerable<string> modTypesToExclude,
+            Dictionary<string, Modification> unknownModifications,
+            bool isContaminant, string rnaDbLocation)
+        {
+            RNA result = null;
+            if (xml.Name == "feature")
+            {
+                ParseFeatureEndElement(xml, modTypesToExclude, unknownModifications);
+            }
+            if (xml.Name == "subfeature")
```

Comment:
are there any benefit from changing this if statements to a switch?

Reply:
Same as above, this is the legacy structure. A switch would likely be an improvement